.buildkite/pipeline.yml

```
@@ -44,15 +44,6 @@ steps:
             - SAUCE_USERNAME
             - SAUCE_ACCESS_KEY
 
-  - label: "🔒 Snyk Security Check"
-    key: "snyk"
-    agents:
-      queue: v1
-    plugins:
-      - ssh://git@github.com/segmentio/snyk-buildkite-plugin#v1.2.0:
-          runtime: npm
-          fail-on: upgradable
-
   - wait: ~
     depends_on: ["snyk", "build"]
```

need to fix this step as well.

```suggestion
    depends_on: ["build"]
```